ts/Accessibility/Components/MenuComponent.ts

```diff
@@ -499,6 +499,7 @@ namespace MenuComponent {
      * @private
      */
     export function compose(
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
This looks like a mistake - we pass ChartClass, but the implementation actually uses the global Chart. @bre1470 do you know what happened here?

Reply:
These are probably overlooked, but I recently tried to fix arguments for webpack and with some of these proper compose functions the tests start to fail. So the arguments should be fixed as long as test results are ok; and for the rest we have to fix the actual implementation to make it work.

Reply:
Tests seems to run fine, so I'll remove 👍